### DISCUSSION TITLE
Use IJulia in a document with knitr blocks

### DISCUSSION THREAD

**Discussion by olynch** [quarto-dev/quarto-cli#4422] [Q&A] [answered]
Is it possible to use IJulia and knitr in the same document? JuliaCall seems to be breaking things, but I want to write use tikz from knitr.

**Comment by olynch** [comment, marked as answer]
Ah, I now see that all computational cells share the same engine.

**Comment by cderv** [comment]
> JuliaCall seems to be breaking things, but I want to write use tikz from knitr.

We are interested either in Quarto or **knitr** repo is this is also not working in R Markdown. **JuliaCall** should be supported in Quarto too and works with **knitr** as expected. If something is not, then this is a bug we have to fix in the stack.

Thanks !
